### ISSUE TITLE
bug: Log is spammed by messages that contain the entire README file.

### ISSUE THREAD

**Issue by odisseus** [sourcegraph/jetbrains#1518]
### Cody Version

baed3871ad6d34b41e2a2c56258ccf4a52ae764e

### IDE Information

IntelliJ IDEA 2022.1 (Community Edition)
Build #IC-221.5080.210, built on April 12, 2022

### Describe the bug

I see 19 warnings containing the exact same text, which seems to be a HTML rendering of `README.md`. This adds up to several thousand lines of logs to scroll through.

### Expected behavior

No warnings or at least shorter messages

### Additional context

```
2024-05-16 05:46:15,064 [1284151]   WARN - #c.i.BundleBase - ignore extra mnemonic in <h1>Sourcegraph Cody + Code Search</h1>

<p>Use Sourcegraph’s AI assistant Cody and Sourcegraph Code Search directly from your JetBrains IDE.</p>

<ul><li><a href="https://about.sourcegraph.com/cody?utm_source=plugins.jetbrains.com&amp;utm_medium=referral">Cody</a> is an AI coding assistant that can write, understand and fix your code. Cody is powered by Sourcegraph’s code graph, and has knowledge of your entire codebase.</li><li>With <a href="https://about.sourcegraph.com/code-search?utm_source=plugins.jetbrains.com&amp;utm_medium=referral">Code Search</a>, you can search code across all your repositories and code hosts—even the code you don’t have locally.</li></ul>

<p><strong>Cody for JetBrains IDEs is in beta. We’d love your <a href="https://github.com/sourcegraph/sourcegraph/discussions/new?category=product-feedback&amp;labels=cody,cody/jetbrains">feedback</a></strong>!</p>

<h2>Cody Features</h2>

<h3>Autocomplete: Cody writes code for you</h3>

<p>Cody autocompletes single lines, or whole functions, in any programming language, configuration file, or documentation. It’s powered by latest instant LLMs for accuracy and performance.</p>

<p><img src="https://storage.googleapis.com/sourcegraph-assets/website/Product%20Animations/AutoCompletion_IntelliJ_SS.jpg" alt="Example of using code autocomplete" /></p>

<h3>Chat: Ask Cody about anything in your codebase</h3>

<p>Cody understands your entire codebase — not just your open files. Ask Cody any question about your code, and it will use Sourcegraph's code graph to answer using knowledge of your codebase.</p>

<p>For example, you can ask Cody:</p>

<ul><li>&quot;How is our app's secret storage implemented on Linux?&quot;</li><li>&quot;Where is the CI config for the web integration tests?&quot;</li><li>&quot;Write a new GraphQL resolver for the AuditLog&quot;</li><li>&quot;Why is the UserConnectionResolver giving an &quot;unknown user&quot; error, and how do I fix it?&quot;</li><li>&quot;Add helpful debug log statements&quot;</li><li>&quot;Make this work&quot; <em>(seriously, it often works—try it!)</em></li></ul>

<p><img src="https://storage.googleapis.com/sourcegraph-assets/website/Product%20Animations/Chat_IntelliJ_SS.jpg" alt="Example of chatting with Cody" /></p>

<h3>Built-in commands</h3>

<p>Cody has quick commands for common actions. Select the commands tab or right-click on a selection of code and choose one of the <code>Ask Cody &gt; ...</code> commands, such as:</p>

<ul><li>Explain code</li><li>Generate unit test</li><li>Smell code</li></ul>

<p><em>We also welcome pull request contributions for new, useful commands!</em></p>

<h3>Swappable LLMs</h3>

<p>Cody supports multiple LLMs including Anthropic Claude, Claude 2, and OpenAI GPT-4/3.5 models, with more coming soon. Cody Pro users can swap models used for chat on-demand.</p>

<h3>Usage</h3>

<p>This plugin works for all Cody plans, including Cody Free, Cody Pro, and Cody Enterprise.</p>

<p>You can find detailed information about Cody's available plans <a href="https://sourcegraph.com/pricing?utm_source=plugins.jetbrains.com&amp;utm_medium=referral">on our website</a>.</p>

<h3>Programming language support</h3>

<p>Cody works for any programming language because it uses LLMs trained on broad data. Cody works great with Python, Go, JavaScript, and TypeScript code.</p>

<h3>Code graph</h3>

<p>Cody is powered by Sourcegraph’s code graph and uses context of your codebase to extend its capabilities. By using context from entire repositories, Cody is able to give more accurate answers and generate idiomatic code.</p>

<p>For example:</p>

<ul><li>Ask Cody to generate an API call. Cody can gather context on your API schema to inform the code it writes.</li><li>Ask Cody to find where in your codebase a specific component is defined. Cody can retrieve and describe the exact files where that component is written.</li><li>Ask Cody questions that require an understanding of multiple files. For example, ask Cody how frontend data is populated in a React app; Cody can find the React component definitions to understand what data is being passed and where it originates.</li></ul>

<h3>Cody Enterprise</h3>

<p>Cody Enterprise is able to retrieve context from your entire remote codebase using Sourcegraph search. This gives Cody the ability to understand and answer questions about all of your code, even the repositories that don't live on your local machine.</p>

<p><a href="https://about.sourcegraph.com/contact/request-info?utm_source=plugins.jetbrains.com&amp;utm_medium=referral">Contact us</a> to set up a trial of Cody Enterprise. If you’re an existing Sourcegraph Enterprise customer, contact your technical advisor.</p>

<h2>Feedback</h2>

<ul><li><a href="https://github.com/sourcegraph/cody/issues">Issue tracker</a></li><li><a href="https://discord.gg/s2qDtYGnAE">Discord chat</a></li><li><a href="https://twitter.com/sourcegraph">Twitter (@sourcegraph)</a></li></ul>

<h2>License</h2>

<p><a href="https://github.com/sourcegraph/jetbrains">Cody's code</a> is open source (Apache License 2.0).</p>

<h2>Code Search features</h2>

<ul><li>Search with Sourcegraph directly from inside the IDE</li><li>Instantly search in all open source repos and your private code</li><li>Peek into any remote repo in the IDE, without checking it out locally</li></ul>

<h2>URL sharing features</h2>

<ul><li>Create URLs to specific code blocks to share them with your teammates</li><li>Open your files on Sourcegraph</li></ul>
```

**Comment by dominiccooney** [comment]
Reportedly fixed in 2022.2: https://youtrack.jetbrains.com/issue/IJPL-7864/Plugin-description-warning-com.intellij.BundleBase-ignore-extra-mnemonic-in...

**Comment by dominiccooney** [comment]
2022.1 is still \~6% of JetBrains users per [https://plugins.jetbrains.com/docs/marketplace/product-versions-in-use-statistics.html](https://plugins.jetbrains.com/docs/marketplace/product-versions-in-use-statistics.html) so it isn't worth dropping support for it to silence this error.

On the other hand, it's not worth ameliorating, it will go away when we roll to 2022.2. So I'm going to close this, there's nothing actionable for us here.
